source/common/umf_pools/disjoint_pool.cpp

```diff
@@ -381,6 +381,15 @@ Slab::~Slab() {
     } catch (MemoryProviderError &e) {
         std::cout << "DisjointPool: error from memory provider: " << e.code
                   << "\n";
+        if (e.code == UMF_RESULT_ERROR_MEMORY_PROVIDER_SPECIFIC) {
+            const char *message = "";
+            int error = 0;
+
+            umfMemoryProviderGetLastNativeError(
+                umfGetLastFailedMemoryProvider(), &message, &error);
+            std::cout << "Native error msg: " << message
```

Comment:
Oh, right. Yeah, ideally we would use some logger here but I don't want to hardcode our UR logger since we plan to move DisjointPool to another repo anyway. There are other places in this file that use `cout` so perhaps we can just ignore the issues on coverity for now?